joomla/joomla-cms
# [5.2] Only register hits on items when GET or POST request

Pull Request for Issue #40476 .

### Summary of Changes
Right now, Joomla registers every call to the URL of a content item as a hit. However, there are cases where for example FeedBurner does HEAD requests to the URL, which isn't really a real request to get the content item. This PR filters the requests to only be registered as a hit when triggered by GET or POST.


### Testing Instructions
Install the testing sample data and visit an article of your choice, for example "Content Component". See the hit counter. Request the same URL with a tool like Postman as a HEAD request.


### Actual result BEFORE applying this Pull Request
The HEAD request is counted as a hit.


### Expected result AFTER applying this Pull Request
The HEAD request is not counted as a hit.


### Link to documentations
Please select:
